backend/api/Areas/Property/Controllers/PropertyController.cs

```diff
@@ -0,0 +1,76 @@
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Policies;
+using Pims.Dal;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+
+namespace Pims.Api.Areas.Property.Controllers
+{
+    /// <summary>
+    /// PropertyController class, provides endpoints for interacting with properties.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("properties")]
+    [Route("v{version:apiVersion}/[area]")]
+    [Route("[area]")]
+    public class PropertyController : ControllerBase
+    {
+        #region Variables
+        private readonly IPimsService _pimsService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a PropertyController class, initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="pimsService"></param>
+        /// <param name="mapper"></param>
+        ///
+        public PropertyController(IPimsService pimsService, IMapper mapper)
+        {
+            _pimsService = pimsService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+        /// <summary>
+        /// Get the property for the specified unique 'pid'.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("{pid}")]
+        [HasPermission(Permissions.PropertyView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Models.Property.PropertyModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "property" })]
+        public IActionResult GetPropertyWithPid(string pid)
```

Comment:
Not sure if we'll need this endpoint presently, but suspect it'll be easier to query for a property from a GIS layer with the pid instead of the PIMS id.